### ISSUE TITLE
issue with typescript

### ISSUE THREAD

**Issue by chrislovecnm** [alexeagle/angular-bazel-example#237]
We are trying to reuse part of your awesome project and are getting a couple of errors:

```
bazel build //js-client/...
ERROR: Analysis of target '//js-client/src/reducers:reducers' failed; build aborted: no such package '@npm//@ngrx/store': yarn_install failed: yarn install v1.9.2
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
$ ngc -p js-client/postinstall.tsconfig.json
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (warning package.json: License should be a valid SPDX license expression
warning Waiting for the other yarn instance to finish (85977)
warning Waiting for the other yarn instance to finish (85978)
warning Waiting for the other yarn instance to finish (85981)
warning Waiting for the other yarn instance to finish (85980)
warning example@0.0.1: License should be a valid SPDX license expression
(node:85979) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
warning "@angular/bazel > tsickle@0.32.1" has incorrect peer dependency "typescript@>=2.4.2 <2.10".
warning "@angular/bazel > @bazel/typescript > tsickle@0.28.0" has incorrect peer dependency "typescript@>=2.4.2 <2.9".
error TS18003: No inputs were found in config file 'tsconfig.json'. Specified 'include' paths were '["node_modules/@ngrx/**/*"]' and 'exclude' paths were '["node_modules/@ngrx/store/migrations/**","node_modules/@ngrx/store/schematics/**","node_modules/@ngrx/store/schematics-core/**"]'.

error Command failed with exit code 1.
)
INFO: Elapsed time: 79.514s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (387 packages loaded)
```

We are trying to update master with the branch that uses k8s rules https://github.com/thelgevold/angular-bazel-example/commits/346cc33c4494d9e10d4b8474204c95df93bad288

Also our project has the nested your project under a js-client directory.  Any ideas on debugging the ts error?

**Comment by chrislovecnm** [comment]
The issue seems to be with the refactoring of the project. 

tl;dr I am trying to move this project to a directory called js-client

**Comment by chrislovecnm** [comment]
https://github.com/chrislovecnm/angular-bazel-example/tree/js-client-refactor

Example of what I am trying to do

**Comment by chrislovecnm** [comment]
figured out the problem.  I needed to add `../` to the post tsconfig file.

**Comment by chrislovecnm** [comment]
https://github.com/chrislovecnm/angular-bazel-example/blob/js-client-refactor/js-client/postinstall.tsconfig.json#L25-L31

provides the example
